Running make checkall fails in generating Gateway API CRDs

After #4686 merge `make checkall` fails

```
$ make checkall
...
Generating Gateway API CRD YAML documents...
error: git cmd = '/usr/bin/git fetch --depth=1 origin v0.5.1-0.20220822185609-d05e07b14156': exit status 128
make: *** [Makefile:238: generate-gateway-yaml] Error 1
```

The reason is that `v0.5.1-0.20220822185609-d05e07b14156` is not a git tag.  It is `go.mod` pseudo-version.

This was worked around by #4686 by forcing version by setting environment variable prior to running `make`

https://github.com/projectcontour/contour/blob/a6036886fb5eceebe9b4dd5a0aa4b16919f6eb12/.github/workflows/prbuild.yaml#L71

so that `Makefile` will skip the grep here:
https://github.com/projectcontour/contour/blob/a6036886fb5eceebe9b4dd5a0aa4b16919f6eb12/Makefile#L10

But when I run `make checkall` locally, I'm not likely setting the environment variable `GATEWAY_API_VERSION=release-0.5 make checkall`.  If not working on Gateway API I might not even know about the current version.

Maybe `Makefile` should fetch the version from `.github/workflows/prbuild.yaml` instead, or maybe turn it around: 
have `Makefile` as the authorative source for the version info by just setting hardcoded version there, like it is now done in `prbuild.yaml`?  

There is second grep for `go.mod` in https://github.com/projectcontour/contour/blob/main/test/scripts/run-gateway-conformance.sh which likely has the same problem.

--- a/.github/workflows/prbuild.yaml
+++ b/.github/workflows/prbuild.yaml
@@ -67,8 +67,6 @@ jobs:
           ./hack/actions/install-kubernetes-toolchain.sh $GITHUB_WORKSPACE/bin
           echo "$GITHUB_WORKSPACE/bin" >> $GITHUB_PATH
       - name: generate
-        env:
-          GATEWAY_API_VERSION: "release-0.5"
         run: |
           make generate lint-yamllint lint-flags
           ./hack/actions/check-uncommitted-codegen.sh
@@ -348,7 +346,6 @@ jobs:
       - name: Gateway API conformance tests
         env:
           LOAD_PREBUILT_IMAGE: "true"
-          GATEWAY_API_VERSION: "release-0.5"
         run: |
           export CONTOUR_E2E_IMAGE="ghcr.io/projectcontour/contour:$(ls ./image/contour-*.tar | sed -E 's/.*contour-(.*).tar/\1/')"
           make gateway-conformance

--- a/changelogs/unreleased/4755-skriss-small.md
+++ b/changelogs/unreleased/4755-skriss-small.md
@@ -1,0 +1,1 @@
+Updates to Gateway API v0.5.1.

--- a/examples/gateway/00-crds.yaml
+++ b/examples/gateway/00-crds.yaml
@@ -3,7 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: gatewayclasses.gateway.networking.k8s.io
@@ -434,7 +434,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: gateways.gateway.networking.k8s.io
@@ -1852,7 +1852,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: httproutes.gateway.networking.k8s.io
@@ -2384,7 +2384,7 @@ spec:
                                             implementations must ensure that unknown
                                             values will not cause a crash. \n Unknown
                                             values here must result in the implementation
-                                            setting the Attached Condition for the
+                                            setting the Accepted Condition for the
                                             Route to `status: False`, with a Reason
                                             of `UnsupportedValue`. \n <gateway:experimental>"
                                           enum:
@@ -2411,7 +2411,7 @@ spec:
                                         values may be added to this enum, implementations
                                         must ensure that unknown values will not cause
                                         a crash. \n Unknown values here must result
-                                        in the implementation setting the Attached
+                                        in the implementation setting the Accepted
                                         Condition for the Route to `status: False`,
                                         with a Reason of `UnsupportedValue`."
                                       enum:
@@ -2426,7 +2426,7 @@ spec:
                                         implementations must ensure that unknown values
                                         will not cause a crash. \n Unknown values
                                         here must result in the implementation setting
-                                        the Attached Condition for the Route to `status:
+                                        the Accepted Condition for the Route to `status:
                                         False`, with a Reason of `UnsupportedValue`."
                                       enum:
                                       - 301
@@ -2462,7 +2462,7 @@ spec:
                                     Note that values may be added to this enum, implementations
                                     must ensure that unknown values will not cause
                                     a crash. \n Unknown values here must result in
-                                    the implementation setting the Attached Condition
+                                    the implementation setting the Accepted Condition
                                     for the Route to `status: False`, with a Reason
                                     of `UnsupportedValue`. \n "
                                   enum:
@@ -2522,7 +2522,7 @@ spec:
                                             implementations must ensure that unknown
                                             values will not cause a crash. \n Unknown
                                             values here must result in the implementation
-                                            setting the Attached Condition for the
+                                            setting the Accepted Condition for the
                                             Route to `status: False`, with a Reason
                                             of `UnsupportedValue`. \n <gateway:experimental>"
                                           enum:
@@ -2895,7 +2895,7 @@ spec:
                                       to this enum, implementations must ensure that
                                       unknown values will not cause a crash. \n Unknown
                                       values here must result in the implementation
-                                      setting the Attached Condition for the Route
+                                      setting the Accepted Condition for the Route
                                       to `status: False`, with a Reason of `UnsupportedValue`.
                                       \n <gateway:experimental>"
                                     enum:
@@ -2922,7 +2922,7 @@ spec:
                                   to this enum, implementations must ensure that unknown
                                   values will not cause a crash. \n Unknown values
                                   here must result in the implementation setting the
-                                  Attached Condition for the Route to `status: False`,
+                                  Accepted Condition for the Route to `status: False`,
                                   with a Reason of `UnsupportedValue`."
                                 enum:
                                 - http
@@ -2935,7 +2935,7 @@ spec:
                                   values may be added to this enum, implementations
                                   must ensure that unknown values will not cause a
                                   crash. \n Unknown values here must result in the
-                                  implementation setting the Attached Condition for
+                                  implementation setting the Accepted Condition for
                                   the Route to `status: False`, with a Reason of `UnsupportedValue`."
                                 enum:
                                 - 301
@@ -2969,7 +2969,7 @@ spec:
                               that values may be added to this enum, implementations
                               must ensure that unknown values will not cause a crash.
                               \n Unknown values here must result in the implementation
-                              setting the Attached Condition for the Route to `status:
+                              setting the Accepted Condition for the Route to `status:
                               False`, with a Reason of `UnsupportedValue`. \n "
                             enum:
                             - RequestHeaderModifier
@@ -3024,7 +3024,7 @@ spec:
                                       to this enum, implementations must ensure that
                                       unknown values will not cause a crash. \n Unknown
                                       values here must result in the implementation
-                                      setting the Attached Condition for the Route
+                                      setting the Accepted Condition for the Route
                                       to `status: False`, with a Reason of `UnsupportedValue`.
                                       \n <gateway:experimental>"
                                     enum:
@@ -3987,7 +3987,7 @@ spec:
                                             implementations must ensure that unknown
                                             values will not cause a crash. \n Unknown
                                             values here must result in the implementation
-                                            setting the Attached Condition for the
+                                            setting the Accepted Condition for the
                                             Route to `status: False`, with a Reason
                                             of `UnsupportedValue`. \n <gateway:experimental>"
                                           enum:
@@ -4014,7 +4014,7 @@ spec:
                                         values may be added to this enum, implementations
                                         must ensure that unknown values will not cause
                                         a crash. \n Unknown values here must result
-                                        in the implementation setting the Attached
+                                        in the implementation setting the Accepted
                                         Condition for the Route to `status: False`,
                                         with a Reason of `UnsupportedValue`."
                                       enum:
@@ -4029,7 +4029,7 @@ spec:
                                         implementations must ensure that unknown values
                                         will not cause a crash. \n Unknown values
                                         here must result in the implementation setting
-                                        the Attached Condition for the Route to `status:
+                                        the Accepted Condition for the Route to `status:
                                         False`, with a Reason of `UnsupportedValue`."
                                       enum:
                                       - 301
@@ -4065,7 +4065,7 @@ spec:
                                     Note that values may be added to this enum, implementations
                                     must ensure that unknown values will not cause
                                     a crash. \n Unknown values here must result in
-                                    the implementation setting the Attached Condition
+                                    the implementation setting the Accepted Condition
                                     for the Route to `status: False`, with a Reason
                                     of `UnsupportedValue`. \n "
                                   enum:
@@ -4125,7 +4125,7 @@ spec:
                                             implementations must ensure that unknown
                                             values will not cause a crash. \n Unknown
                                             values here must result in the implementation
-                                            setting the Attached Condition for the
+                                            setting the Accepted Condition for the
                                             Route to `status: False`, with a Reason
                                             of `UnsupportedValue`. \n <gateway:experimental>"
                                           enum:
@@ -4497,7 +4497,7 @@ spec:
                                       to this enum, implementations must ensure that
                                       unknown values will not cause a crash. \n Unknown
                                       values here must result in the implementation
-                                      setting the Attached Condition for the Route
+                                      setting the Accepted Condition for the Route
                                       to `status: False`, with a Reason of `UnsupportedValue`.
                                       \n <gateway:experimental>"
                                     enum:
@@ -4524,7 +4524,7 @@ spec:
                                   to this enum, implementations must ensure that unknown
                                   values will not cause a crash. \n Unknown values
                                   here must result in the implementation setting the
-                                  Attached Condition for the Route to `status: False`,
+                                  Accepted Condition for the Route to `status: False`,
                                   with a Reason of `UnsupportedValue`."
                                 enum:
                                 - http
@@ -4537,7 +4537,7 @@ spec:
                                   values may be added to this enum, implementations
                                   must ensure that unknown values will not cause a
                                   crash. \n Unknown values here must result in the
-                                  implementation setting the Attached Condition for
+                                  implementation setting the Accepted Condition for
                                   the Route to `status: False`, with a Reason of `UnsupportedValue`."
                                 enum:
                                 - 301
@@ -4571,7 +4571,7 @@ spec:
                               that values may be added to this enum, implementations
                               must ensure that unknown values will not cause a crash.
                               \n Unknown values here must result in the implementation
-                              setting the Attached Condition for the Route to `status:
+                              setting the Accepted Condition for the Route to `status:
                               False`, with a Reason of `UnsupportedValue`. \n "
                             enum:
                             - RequestHeaderModifier
@@ -4626,7 +4626,7 @@ spec:
                                       to this enum, implementations must ensure that
                                       unknown values will not cause a crash. \n Unknown
                                       values here must result in the implementation
-                                      setting the Attached Condition for the Route
+                                      setting the Accepted Condition for the Route
                                       to `status: False`, with a Reason of `UnsupportedValue`.
                                       \n <gateway:experimental>"
                                     enum:
@@ -5085,7 +5085,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: referencegrants.gateway.networking.k8s.io
@@ -5235,7 +5235,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: referencepolicies.gateway.networking.k8s.io
@@ -5390,7 +5390,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: tcproutes.gateway.networking.k8s.io
@@ -5884,7 +5884,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: tlsroutes.gateway.networking.k8s.io
@@ -6427,7 +6427,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: udproutes.gateway.networking.k8s.io

--- a/examples/gateway/01-admission_webhook.yaml
+++ b/examples/gateway/01-admission_webhook.yaml
@@ -56,7 +56,7 @@ spec:
     spec:
       containers:
         - name: webhook
-          image: gcr.io/k8s-staging-gateway-api/admission-server:v0.5.0
+          image: gcr.io/k8s-staging-gateway-api/admission-server:v0.5.1
           imagePullPolicy: Always
           args:
             - -logtostderr

--- a/examples/render/contour-gateway-provisioner.yaml
+++ b/examples/render/contour-gateway-provisioner.yaml
@@ -4662,7 +4662,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: gatewayclasses.gateway.networking.k8s.io
@@ -5093,7 +5093,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: gateways.gateway.networking.k8s.io
@@ -6511,7 +6511,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: httproutes.gateway.networking.k8s.io
@@ -7043,7 +7043,7 @@ spec:
                                             implementations must ensure that unknown
                                             values will not cause a crash. \n Unknown
                                             values here must result in the implementation
-                                            setting the Attached Condition for the
+                                            setting the Accepted Condition for the
                                             Route to `status: False`, with a Reason
                                             of `UnsupportedValue`. \n <gateway:experimental>"
                                           enum:
@@ -7070,7 +7070,7 @@ spec:
                                         values may be added to this enum, implementations
                                         must ensure that unknown values will not cause
                                         a crash. \n Unknown values here must result
-                                        in the implementation setting the Attached
+                                        in the implementation setting the Accepted
                                         Condition for the Route to `status: False`,
                                         with a Reason of `UnsupportedValue`."
                                       enum:
@@ -7085,7 +7085,7 @@ spec:
                                         implementations must ensure that unknown values
                                         will not cause a crash. \n Unknown values
                                         here must result in the implementation setting
-                                        the Attached Condition for the Route to `status:
+                                        the Accepted Condition for the Route to `status:
                                         False`, with a Reason of `UnsupportedValue`."
                                       enum:
                                       - 301
@@ -7121,7 +7121,7 @@ spec:
                                     Note that values may be added to this enum, implementations
                                     must ensure that unknown values will not cause
                                     a crash. \n Unknown values here must result in
-                                    the implementation setting the Attached Condition
+                                    the implementation setting the Accepted Condition
                                     for the Route to `status: False`, with a Reason
                                     of `UnsupportedValue`. \n "
                                   enum:
@@ -7181,7 +7181,7 @@ spec:
                                             implementations must ensure that unknown
                                             values will not cause a crash. \n Unknown
                                             values here must result in the implementation
-                                            setting the Attached Condition for the
+                                            setting the Accepted Condition for the
                                             Route to `status: False`, with a Reason
                                             of `UnsupportedValue`. \n <gateway:experimental>"
                                           enum:
@@ -7554,7 +7554,7 @@ spec:
                                       to this enum, implementations must ensure that
                                       unknown values will not cause a crash. \n Unknown
                                       values here must result in the implementation
-                                      setting the Attached Condition for the Route
+                                      setting the Accepted Condition for the Route
                                       to `status: False`, with a Reason of `UnsupportedValue`.
                                       \n <gateway:experimental>"
                                     enum:
@@ -7581,7 +7581,7 @@ spec:
                                   to this enum, implementations must ensure that unknown
                                   values will not cause a crash. \n Unknown values
                                   here must result in the implementation setting the
-                                  Attached Condition for the Route to `status: False`,
+                                  Accepted Condition for the Route to `status: False`,
                                   with a Reason of `UnsupportedValue`."
                                 enum:
                                 - http
@@ -7594,7 +7594,7 @@ spec:
                                   values may be added to this enum, implementations
                                   must ensure that unknown values will not cause a
                                   crash. \n Unknown values here must result in the
-                                  implementation setting the Attached Condition for
+                                  implementation setting the Accepted Condition for
                                   the Route to `status: False`, with a Reason of `UnsupportedValue`."
                                 enum:
                                 - 301
@@ -7628,7 +7628,7 @@ spec:
                               that values may be added to this enum, implementations
                               must ensure that unknown values will not cause a crash.
                               \n Unknown values here must result in the implementation
-                              setting the Attached Condition for the Route to `status:
+                              setting the Accepted Condition for the Route to `status:
                               False`, with a Reason of `UnsupportedValue`. \n "
                             enum:
                             - RequestHeaderModifier
@@ -7683,7 +7683,7 @@ spec:
                                       to this enum, implementations must ensure that
                                       unknown values will not cause a crash. \n Unknown
                                       values here must result in the implementation
-                                      setting the Attached Condition for the Route
+                                      setting the Accepted Condition for the Route
                                       to `status: False`, with a Reason of `UnsupportedValue`.
                                       \n <gateway:experimental>"
                                     enum:
@@ -8646,7 +8646,7 @@ spec:
                                             implementations must ensure that unknown
                                             values will not cause a crash. \n Unknown
                                             values here must result in the implementation
-                                            setting the Attached Condition for the
+                                            setting the Accepted Condition for the
                                             Route to `status: False`, with a Reason
                                             of `UnsupportedValue`. \n <gateway:experimental>"
                                           enum:
@@ -8673,7 +8673,7 @@ spec:
                                         values may be added to this enum, implementations
                                         must ensure that unknown values will not cause
                                         a crash. \n Unknown values here must result
-                                        in the implementation setting the Attached
+                                        in the implementation setting the Accepted
                                         Condition for the Route to `status: False`,
                                         with a Reason of `UnsupportedValue`."
                                       enum:
@@ -8688,7 +8688,7 @@ spec:
                                         implementations must ensure that unknown values
                                         will not cause a crash. \n Unknown values
                                         here must result in the implementation setting
-                                        the Attached Condition for the Route to `status:
+                                        the Accepted Condition for the Route to `status:
                                         False`, with a Reason of `UnsupportedValue`."
                                       enum:
                                       - 301
@@ -8724,7 +8724,7 @@ spec:
                                     Note that values may be added to this enum, implementations
                                     must ensure that unknown values will not cause
                                     a crash. \n Unknown values here must result in
-                                    the implementation setting the Attached Condition
+                                    the implementation setting the Accepted Condition
                                     for the Route to `status: False`, with a Reason
                                     of `UnsupportedValue`. \n "
                                   enum:
@@ -8784,7 +8784,7 @@ spec:
                                             implementations must ensure that unknown
                                             values will not cause a crash. \n Unknown
                                             values here must result in the implementation
-                                            setting the Attached Condition for the
+                                            setting the Accepted Condition for the
                                             Route to `status: False`, with a Reason
                                             of `UnsupportedValue`. \n <gateway:experimental>"
                                           enum:
@@ -9156,7 +9156,7 @@ spec:
                                       to this enum, implementations must ensure that
                                       unknown values will not cause a crash. \n Unknown
                                       values here must result in the implementation
-                                      setting the Attached Condition for the Route
+                                      setting the Accepted Condition for the Route
                                       to `status: False`, with a Reason of `UnsupportedValue`.
                                       \n <gateway:experimental>"
                                     enum:
@@ -9183,7 +9183,7 @@ spec:
                                   to this enum, implementations must ensure that unknown
                                   values will not cause a crash. \n Unknown values
                                   here must result in the implementation setting the
-                                  Attached Condition for the Route to `status: False`,
+                                  Accepted Condition for the Route to `status: False`,
                                   with a Reason of `UnsupportedValue`."
                                 enum:
                                 - http
@@ -9196,7 +9196,7 @@ spec:
                                   values may be added to this enum, implementations
                                   must ensure that unknown values will not cause a
                                   crash. \n Unknown values here must result in the
-                                  implementation setting the Attached Condition for
+                                  implementation setting the Accepted Condition for
                                   the Route to `status: False`, with a Reason of `UnsupportedValue`."
                                 enum:
                                 - 301
@@ -9230,7 +9230,7 @@ spec:
                               that values may be added to this enum, implementations
                               must ensure that unknown values will not cause a crash.
                               \n Unknown values here must result in the implementation
-                              setting the Attached Condition for the Route to `status:
+                              setting the Accepted Condition for the Route to `status:
                               False`, with a Reason of `UnsupportedValue`. \n "
                             enum:
                             - RequestHeaderModifier
@@ -9285,7 +9285,7 @@ spec:
                                       to this enum, implementations must ensure that
                                       unknown values will not cause a crash. \n Unknown
                                       values here must result in the implementation
-                                      setting the Attached Condition for the Route
+                                      setting the Accepted Condition for the Route
                                       to `status: False`, with a Reason of `UnsupportedValue`.
                                       \n <gateway:experimental>"
                                     enum:
@@ -9744,7 +9744,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: referencegrants.gateway.networking.k8s.io
@@ -9894,7 +9894,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: referencepolicies.gateway.networking.k8s.io
@@ -10049,7 +10049,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: tcproutes.gateway.networking.k8s.io
@@ -10543,7 +10543,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: tlsroutes.gateway.networking.k8s.io
@@ -11086,7 +11086,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: udproutes.gateway.networking.k8s.io
@@ -11640,7 +11640,7 @@ spec:
     spec:
       containers:
         - name: webhook
-          image: gcr.io/k8s-staging-gateway-api/admission-server:v0.5.0
+          image: gcr.io/k8s-staging-gateway-api/admission-server:v0.5.1
           imagePullPolicy: Always
           args:
             - -logtostderr

--- a/examples/render/contour-gateway.yaml
+++ b/examples/render/contour-gateway.yaml
@@ -5368,7 +5368,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: gatewayclasses.gateway.networking.k8s.io
@@ -5799,7 +5799,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: gateways.gateway.networking.k8s.io
@@ -7217,7 +7217,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: httproutes.gateway.networking.k8s.io
@@ -7749,7 +7749,7 @@ spec:
                                             implementations must ensure that unknown
                                             values will not cause a crash. \n Unknown
                                             values here must result in the implementation
-                                            setting the Attached Condition for the
+                                            setting the Accepted Condition for the
                                             Route to `status: False`, with a Reason
                                             of `UnsupportedValue`. \n <gateway:experimental>"
                                           enum:
@@ -7776,7 +7776,7 @@ spec:
                                         values may be added to this enum, implementations
                                         must ensure that unknown values will not cause
                                         a crash. \n Unknown values here must result
-                                        in the implementation setting the Attached
+                                        in the implementation setting the Accepted
                                         Condition for the Route to `status: False`,
                                         with a Reason of `UnsupportedValue`."
                                       enum:
@@ -7791,7 +7791,7 @@ spec:
                                         implementations must ensure that unknown values
                                         will not cause a crash. \n Unknown values
                                         here must result in the implementation setting
-                                        the Attached Condition for the Route to `status:
+                                        the Accepted Condition for the Route to `status:
                                         False`, with a Reason of `UnsupportedValue`."
                                       enum:
                                       - 301
@@ -7827,7 +7827,7 @@ spec:
                                     Note that values may be added to this enum, implementations
                                     must ensure that unknown values will not cause
                                     a crash. \n Unknown values here must result in
-                                    the implementation setting the Attached Condition
+                                    the implementation setting the Accepted Condition
                                     for the Route to `status: False`, with a Reason
                                     of `UnsupportedValue`. \n "
                                   enum:
@@ -7887,7 +7887,7 @@ spec:
                                             implementations must ensure that unknown
                                             values will not cause a crash. \n Unknown
                                             values here must result in the implementation
-                                            setting the Attached Condition for the
+                                            setting the Accepted Condition for the
                                             Route to `status: False`, with a Reason
                                             of `UnsupportedValue`. \n <gateway:experimental>"
                                           enum:
@@ -8260,7 +8260,7 @@ spec:
                                       to this enum, implementations must ensure that
                                       unknown values will not cause a crash. \n Unknown
                                       values here must result in the implementation
-                                      setting the Attached Condition for the Route
+                                      setting the Accepted Condition for the Route
                                       to `status: False`, with a Reason of `UnsupportedValue`.
                                       \n <gateway:experimental>"
                                     enum:
@@ -8287,7 +8287,7 @@ spec:
                                   to this enum, implementations must ensure that unknown
                                   values will not cause a crash. \n Unknown values
                                   here must result in the implementation setting the
-                                  Attached Condition for the Route to `status: False`,
+                                  Accepted Condition for the Route to `status: False`,
                                   with a Reason of `UnsupportedValue`."
                                 enum:
                                 - http
@@ -8300,7 +8300,7 @@ spec:
                                   values may be added to this enum, implementations
                                   must ensure that unknown values will not cause a
                                   crash. \n Unknown values here must result in the
-                                  implementation setting the Attached Condition for
+                                  implementation setting the Accepted Condition for
                                   the Route to `status: False`, with a Reason of `UnsupportedValue`."
                                 enum:
                                 - 301
@@ -8334,7 +8334,7 @@ spec:
                               that values may be added to this enum, implementations
                               must ensure that unknown values will not cause a crash.
                               \n Unknown values here must result in the implementation
-                              setting the Attached Condition for the Route to `status:
+                              setting the Accepted Condition for the Route to `status:
                               False`, with a Reason of `UnsupportedValue`. \n "
                             enum:
                             - RequestHeaderModifier
@@ -8389,7 +8389,7 @@ spec:
                                       to this enum, implementations must ensure that
                                       unknown values will not cause a crash. \n Unknown
                                       values here must result in the implementation
-                                      setting the Attached Condition for the Route
+                                      setting the Accepted Condition for the Route
                                       to `status: False`, with a Reason of `UnsupportedValue`.
                                       \n <gateway:experimental>"
                                     enum:
@@ -9352,7 +9352,7 @@ spec:
                                             implementations must ensure that unknown
                                             values will not cause a crash. \n Unknown
                                             values here must result in the implementation
-                                            setting the Attached Condition for the
+                                            setting the Accepted Condition for the
                                             Route to `status: False`, with a Reason
                                             of `UnsupportedValue`. \n <gateway:experimental>"
                                           enum:
@@ -9379,7 +9379,7 @@ spec:
                                         values may be added to this enum, implementations
                                         must ensure that unknown values will not cause
                                         a crash. \n Unknown values here must result
-                                        in the implementation setting the Attached
+                                        in the implementation setting the Accepted
                                         Condition for the Route to `status: False`,
                                         with a Reason of `UnsupportedValue`."
                                       enum:
@@ -9394,7 +9394,7 @@ spec:
                                         implementations must ensure that unknown values
                                         will not cause a crash. \n Unknown values
                                         here must result in the implementation setting
-                                        the Attached Condition for the Route to `status:
+                                        the Accepted Condition for the Route to `status:
                                         False`, with a Reason of `UnsupportedValue`."
                                       enum:
                                       - 301
@@ -9430,7 +9430,7 @@ spec:
                                     Note that values may be added to this enum, implementations
                                     must ensure that unknown values will not cause
                                     a crash. \n Unknown values here must result in
-                                    the implementation setting the Attached Condition
+                                    the implementation setting the Accepted Condition
                                     for the Route to `status: False`, with a Reason
                                     of `UnsupportedValue`. \n "
                                   enum:
@@ -9490,7 +9490,7 @@ spec:
                                             implementations must ensure that unknown
                                             values will not cause a crash. \n Unknown
                                             values here must result in the implementation
-                                            setting the Attached Condition for the
+                                            setting the Accepted Condition for the
                                             Route to `status: False`, with a Reason
                                             of `UnsupportedValue`. \n <gateway:experimental>"
                                           enum:
@@ -9862,7 +9862,7 @@ spec:
                                       to this enum, implementations must ensure that
                                       unknown values will not cause a crash. \n Unknown
                                       values here must result in the implementation
-                                      setting the Attached Condition for the Route
+                                      setting the Accepted Condition for the Route
                                       to `status: False`, with a Reason of `UnsupportedValue`.
                                       \n <gateway:experimental>"
                                     enum:
@@ -9889,7 +9889,7 @@ spec:
                                   to this enum, implementations must ensure that unknown
                                   values will not cause a crash. \n Unknown values
                                   here must result in the implementation setting the
-                                  Attached Condition for the Route to `status: False`,
+                                  Accepted Condition for the Route to `status: False`,
                                   with a Reason of `UnsupportedValue`."
                                 enum:
                                 - http
@@ -9902,7 +9902,7 @@ spec:
                                   values may be added to this enum, implementations
                                   must ensure that unknown values will not cause a
                                   crash. \n Unknown values here must result in the
-                                  implementation setting the Attached Condition for
+                                  implementation setting the Accepted Condition for
                                   the Route to `status: False`, with a Reason of `UnsupportedValue`."
                                 enum:
                                 - 301
@@ -9936,7 +9936,7 @@ spec:
                               that values may be added to this enum, implementations
                               must ensure that unknown values will not cause a crash.
                               \n Unknown values here must result in the implementation
-                              setting the Attached Condition for the Route to `status:
+                              setting the Accepted Condition for the Route to `status:
                               False`, with a Reason of `UnsupportedValue`. \n "
                             enum:
                             - RequestHeaderModifier
@@ -9991,7 +9991,7 @@ spec:
                                       to this enum, implementations must ensure that
                                       unknown values will not cause a crash. \n Unknown
                                       values here must result in the implementation
-                                      setting the Attached Condition for the Route
+                                      setting the Accepted Condition for the Route
                                       to `status: False`, with a Reason of `UnsupportedValue`.
                                       \n <gateway:experimental>"
                                     enum:
@@ -10450,7 +10450,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: referencegrants.gateway.networking.k8s.io
@@ -10600,7 +10600,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: referencepolicies.gateway.networking.k8s.io
@@ -10755,7 +10755,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: tcproutes.gateway.networking.k8s.io
@@ -11249,7 +11249,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: tlsroutes.gateway.networking.k8s.io
@@ -11792,7 +11792,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1086
-    gateway.networking.k8s.io/bundle-version: v0.5.0
+    gateway.networking.k8s.io/bundle-version: v0.5.1
     gateway.networking.k8s.io/channel: experimental
   creationTimestamp: null
   name: udproutes.gateway.networking.k8s.io
@@ -12346,7 +12346,7 @@ spec:
     spec:
       containers:
         - name: webhook
-          image: gcr.io/k8s-staging-gateway-api/admission-server:v0.5.0
+          image: gcr.io/k8s-staging-gateway-api/admission-server:v0.5.1
           imagePullPolicy: Always
           args:
             - -logtostderr

--- a/go.mod
+++ b/go.mod
@@ -41,6 +41,6 @@ require (
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
 	sigs.k8s.io/controller-runtime v0.12.1
 	sigs.k8s.io/controller-tools v0.7.0
-	sigs.k8s.io/gateway-api v0.5.1-0.20220822185609-d05e07b14156
+	sigs.k8s.io/gateway-api v0.5.1
 	sigs.k8s.io/kustomize/kyaml v0.10.17
 )

